Gemfile.jruby-3.1.lock.release

```diff
@@ -0,0 +1,1018 @@
+PATH
+  remote: logstash-core-plugin-api
+  specs:
+    logstash-core-plugin-api (2.1.16-java)
+      logstash-core (= 8.17.2)
```

Comment:
maybe you would like to update the version?

Reply:
there is a follow up PR after this